### PR TITLE
Rename package, remove egg-info

### PR DESCRIPTION
**Pull request by max-zilla**
This should replace all references to pygeotemporal with pygeostreams. Also got rid of a few unnecessary temp files.

If you have already cloned the repo and want to update that instead of a fresh clone, you can use `git remote set-url origin` to point to a new URL.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,7 +5,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
-## [Unreleased]
+## 0.2.0
```
Comment (lmarini) [inline CHANGELOG.md:8]: we have 3.0.0 tag already so this should probably be 4.0.0

File: docs/source/conf.py [hunk 2]
```diff
@@ -48,7 +48,7 @@
 master_doc = 'index'
 
 # General information about the project.
-project = u'Pygeotemporal'
+project = u'Pygeostreams'
 copyright = u'2017, Luigi Marini, Yan Zhao, Marcus Slavenas'
```
Comment (lmarini) [inline docs/source/conf.py:52]: add your name?

File: setup.py [hunk 3]
```diff
@@ -1,7 +1,7 @@
 from distutils.core import setup
 
-setup(name='pygeotemporal',
-      version='0.1.0',
-      url='https://opensource.ncsa.illinois.edu/bitbucket/projects/GEOD/repos/pygeotemporal/browse',
-      packages=['pygeotemporal'],
+setup(name='pygeostreams',
+      version='0.2.0',
```
Comment (lmarini) [inline setup.py:4]: 4.0.0

**Review by lmarini — CHANGES_REQUESTED** [review]

File: setup.py (hunk 3 above)
Comment (max-zilla) [reply setup.py]: made this change throughout and added myself to docs copyright

**Comment by lmarini** [comment]
When creating a pip virtual env I got this error under python 3.8:

```ERROR: Could not find a version that satisfies the requirement pytest-capturelog==0.7```

**Review by lmarini — APPROVED** [review]

**Pull request merged**